src/barchart/BarChart.jsx

```diff
@@ -57,9 +55,19 @@ module.exports = React.createClass({
         .rangeRoundBands([0, props.width - sideMargins], 0.1);
 
     var trans = `translate(${ margins.left },${ margins.top })`;
-    
+
     return (
-      <Chart width={props.width} height={props.height} title={props.title}>
+      <Chart
+        viewBox={props.viewBox}
+        legend={props.legend}
+        data={props.data}
+        margins={props.margins}
+        colors={props.colors}
+        colorAccessor={props.colorAccessor}
```

Comment:
I thought I had done it already when I first added `colorAccessor` to all the charts. I only hadn't done it to `BarChart` because at that time `BarChart` wasn't even using `colors`